### PR TITLE
Add Authentication page with login and Nickname

### PR DESCRIPTION
**Pull request by wdcroslen**

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    response.getWriter().println("<a href=\"" + logoutUrl + "\"><button>Logout</button/></a>");
+
+    }
+
+
+   private String getUserNickname(String id) {
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    Query query =
+        new Query("UserInfo");
+            // .setFilter(new Query.FilterPredicate("id", Query.FilterOperator.EQUAL, id));
```
Comment (jiaxinz-g) [inline portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java]: Let's remove commented code.

File: portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java [hunk 2]
```diff
@@ -55,34 +46,27 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
...
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     // Get the input from the form.
-    String text = getParameter(request, "text-input", "oof");
-    long timestamp = System.currentTimeMillis();
+    String text = getParameter(request, "text-input", "");
     
+    long timestamp = System.currentTimeMillis();
+    //FIXME : check if text input = 3 and  return
```
Comment (jiaxinz-g) [inline portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java:65]: Is this still relevant? If not, please remove.

**Review by jiaxinz-g — APPROVED** [review]
Looks good to me, please wait for other reviewers to take a look.

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -25,16 +25,18 @@ <h1>Fact method</h1>
       <div id="fact-container"></div>
       <p>Click <a href="aboutMe.html">here</a> to learn more about me!</p>
       <!-- button to view data servlet-->
-      <!-- <a href="data"> <button>DATA</button/></a> -->
+      <a href="login"> <button onclick = "unHideComment()">Log in</button/></a>
```
Comment (hilakey) [inline portfolio/src/main/webapp/index.html:28]: I like the way you utilized the onclick! It's such a cool way you implemented it. Everything else loos good to me.

**Review by hilakey — APPROVED** [review]

File: portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java (hunk 1 above)
Comment (wdcroslen) [reply portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java]: Done!

File: portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java (hunk 2 above)
Comment (wdcroslen) [reply portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java:65]: Thank you for catching this, I have removed  it.

File: portfolio/src/main/webapp/index.html (hunk 3 above)
Comment (wdcroslen) [reply portfolio/src/main/webapp/index.html:28]: Thank you!

**Pull request merged**